core/src/test/java/nl/nn/adapterframework/xml/XmlWriterTest.java

```diff
@@ -62,8 +62,8 @@ public void testNoLexicalHandling() throws Exception {
 		XmlWriter xmlWriter = new XmlWriter();
 		
 		InputSource inputSource = new InputSource(new StringReader(input));
-		XMLReader xmlReader = XmlUtils.getXMLReader(true, true);
-		xmlReader.setContentHandler(xmlWriter);
+		XMLReader xmlReader = XmlUtils.getXMLReader(xmlWriter);
+		xmlReader.setProperty("http://xml.org/sax/properties/lexical-handler", null);
```

Comment:
Maybe you could add a small description (comment) why you're explicitly turning off the lexical-handler